_data/simple-icons.json

```
@@ -11857,6 +11857,11 @@
             "hex": "2480E6",
             "source": "https://processwire.com"
         },
+        {
+            "title": "Procreate",
+            "hex": "E559D7",
+            "source": "https://procreate.art"
+        },    
```

Here the linter is failing due to the extra spaces after the comma:
```suggestion
        {
            "title": "Procreate",
            "hex": "E559D7",
            "source": "https://procreate.art"
        },
```